### PR TITLE
Url import imporvements

### PR DESCRIPTION
**Pull request by sri2k1us**
- Improve UX around URL import
- Display error(s) and success msgs inside dialog
- Update to i18nNext.

- Empty URL:
image

- Duplicate import:
image

- URL import submitted:
image

### CODE REVIEW

File: src/components/data/listing/Listing.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ import constants from "../../../constants";
 import DEPagination from "components/utils/DEPagination";
 import ResourceTypes from "components/models/ResourceTypes";
 import isQueryLoading from "components/utils/isQueryLoading";
-import URLImportDialog from "components/URLImportDialog";
+import URLImportDialog from "components/urlImport";
```
Comment (sri2k1us) [inline src/components/data/listing/Listing.js:21]: Updated folder name for consistency (lowercase)

File: src/components/layout/Notifications.js [hunk 2]
```diff
@@ -28,8 +28,11 @@ import {
 import { useTheme } from "@material-ui/core/styles";
 import NotificationsIcon from "@material-ui/icons/Notifications";
 
+const ANALYSIS_EMAIL_TEMPLATE = "analysis_status_change";
+
 function getDisplayMessage(notification) {
-    return notification.type === "data"
+    return notification.type === "data" &&
+        notification["email_template"] !== ANALYSIS_EMAIL_TEMPLATE
         ? notification.subject
```
Comment (sri2k1us) [inline src/components/layout/Notifications.js:36]: exclude URL import notification from this check.

File: src/components/data/listing/Listing.js (hunk 1 above)
Comment (ianmcorvidae) [reply src/components/data/listing/Listing.js:21]: Is this consistent? I thought that we used initial uppercase for components (like `URLImportDialog`) and lowercase for non-component stuff like `isQueryLoading`. I'm not as used to this code, of course, but I'd expect it to be the former name.

**Review by ianmcorvidae — APPROVED** [review]
Looks pretty reasonable to me. I had one comment which may or may not make sense.

File: src/components/data/listing/Listing.js (hunk 1 above)
Comment (sri2k1us) [reply src/components/data/listing/Listing.js:21]: I just renamed the folder not the the file `index.js` that contains the `URLImportDialog`. (folder `URLImportDialog` was renamed to `urlImport `). Folders containing the components should use all lowercase.

File: src/components/urlImport/index.js [hunk 3]
```diff
@@ -247,42 +234,69 @@ const URLImportDialog = (props) => {
...
-                    {fmt(intl, "instructions")}
+                    {t("instructions")}
                 </Typography>
+
                 <URLImportTextField
-                    intl={intl}
                     path={path}
-                    showErrorAnnouncer={showErrorAnnouncer}
+                    successCallback={(msg) => setSubmittedMsg(msg)}
+                    errorCallback={(msg) => setErrorMsg(msg)}
```
Comment (ash3rz) [inline src/components/urlImport/index.js:259]: For some reason, calling these `successCallback` and `errorCallback` confused me.  Might just be me though.

**Review by ash3rz — APPROVED** [review]
LGTM :shipit:

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**